src/test/java/core/basesyntax/service/write/FileWriterImplTest.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.write;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterImplTest {
+    private static final String CORRECT_PATH = "src/test/resources/report_test.csv";
+    private static final String INCORRECT_PATH = "";
+    private static FileWriter fileWriter;
+
+    @BeforeClass
+    public static void setUp() {
+        fileWriter = new FileWriterImpl();
+    }
+
+    @Test
+    public void white_correctPath_Ok() {
+        Assert.assertTrue(fileWriter.write("", CORRECT_PATH));
```

Comment:
let's add a reader to read from file and ensure that content is saved. we can use jdk reader